src/style.css

```diff
@@ -0,0 +1,21 @@
+body {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+.main--heading {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.sub--heading {
+  font-size: 15px;
+  margin-top: 1em;
+  margin-bottom: 1em;
+  font-weight: 400;
+}
+
+.sub--heading a {
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice